src/util/pseudonymDict.js

```diff
@@ -0,0 +1,1459 @@
+  '史賓杜',
+  '史丹',
+  '史丹佛',
+  '史丹尼',
+  '史帝文',
+  '史帝夫',
+  '史都華德',
+  '塔伯',
+  '泰勒',
+  '泰德',
+  '泰倫斯',
+  '希爾保特',
+  '希歐多爾',
+  '托瑪士',
+  '帝福尼',
+  '堤姆',
+  '帝摩斯',
+  '托拜西',
+  '托比',
+  '陶德',
+  '湯姆',
+  '湯尼',
+  '特瑞西',
+  '特洛伊',
+  '杜魯門',
+  '泰勒',
+  '泰倫',
+  '尤裏西斯',
+  '阿普頓',
+  '尤萊亞',
+  '範倫鐵恩',
+  '范倫丁',
+  '佛能',
+  '維克',
+  '維克多',
+  '文森',
+  '維吉爾',
+  '維托',
+  '衛威恩',
+  '維德',
+  '瓦爾克',
+  '瓦爾特',
+  '華德',
+  '華納',
+  '韋恩',
+  '韋勃',
+  '韋伯斯特',
+  '溫德爾',
+  '韋納爾',
+  '韋爾伯',
+  '威爾',
+  '威廉',
+  '威利',
+  '威弗列德',
+  '溫士頓',
+  '伍德洛',
+  '渥茲華斯',
+  '萊特',
+  '偉茲',
+  '賽維爾',
+  '耶魯',
+  '耶呼弟',
+  '約克',
+  '依夫',
+  '紮克利',
+  '紀伯倫',
+  '日傑夫',
+  '劉德華',
+  '金城武',
+];
+export const adjectives = [
+  '忠懇的',
+  '厚道的',
+  '理智的',
+  '忠厚的',
+  '勤勞的',
+  '勇敢的',
+  '勤奮的',
+  '勤儉的',
+  '幽默的',
+  '文雅的',
+  '禮貌的',
+  '斯文的',
+  '開朗的',
+  '直率的',
+  '耿直的',
+  '溫柔的',
+  '賢慧的',
+  '自重自愛的',
+  '有思想的',
+  '大膽的',
+  '體貼的',
+  '精力充沛的',
+  '禮貌的',
+  '內向的',
+  '性情平和的',
+  '謙虛的',
+  '好學上進的',
+  '有見解的',
+  '辦事仔細的',
+  '細心的',
+  '務實的',
+  '善良的',
+  '老實的',
+  '沉靜的',
+  '謙遜的',
+  '勇挑重擔的',
+  '善解人意的',
+  '正直無私的',
+  '好沉思的',
+  '充滿熱情的',
+  '恭順的',
+  '溫和的',
+  '謹慎的',
+  '好心腸的',
+  '一本正經的',
+  '敏銳的',
+  '專心致志的',
+  '有冒險精神的',
+  '愛冒險的',
+  '善於社交的',
+  '體貼的',
+  '生動的',
+  '高大的',
+  '平和的',
+  '含蓄的',
+  '競爭性的',
+  '積極的',
+  '受尊重的',
+  '尊敬的',
+  '強壯的',
+  '有趣的',
+  '忠誠的',
+  '激勵性的',
+  '生氣勃勃的',
+  '果斷的',
+  '意志堅定的',
+  '發起者的',
+  '愉快的',
+  '外向的',
+  '表情豐富的',
+  '適應能力強的',
+  '樂觀的',
+  '快樂的',
+  '開心的',
+  '興奮的',
+  '執著的',
+  '衝動的',
+  '豁達的',
+  '善良堅強的',
+  '捨己為人的',
+  '有能力才幹的',
+  '寬厚的',
+  '有合作精神的',
+  '有獨創精神的',
+  '穩健的',
+  '廉潔的',
+  '不怕艱險的',
+  '主動活躍的',
+  '忠心耿耿的',
+  '富創造力的',
+  '守信的',
+  '客觀實際的',
+  '孜孜不倦的',
+  '嚴以律已的',
+  '愛恨分明的',
+  '適應性強的',
+  '寬已待人的',
+  '有衝勁的',
+  '忠誠的',
+  '目明確的',
+  '大公無私的',
+  '有進取心的',
+  '有信心的',
+  '有奉獻精神的',
+  '意志堅強的',
+  '一絲不苟的',
+  '平易近人的',
+  '有雄心壯志的',
+  '認真自覺的',
+  '可靠的',
+  '直率真誠的',
+  '不屈不撓的',
+  '善交際的',
+  '高風亮節的',
+  '和藹可親的',
+  '虛心的',
+  '老練的',
+  '儉樸的',
+  '嚴守時刻的',
+  '耐性的',
+  '獨立的',
+  '友善的',
+  '令人高興的',
+  '自立的',
+  '深沉的',
+  '適應力強的',
+  '音樂性的',
+  '熱情的',
+  '活潑的',
+  '反應快的',
+  '敏捷的',
+  '率直的',
+  '偉大的',
+  '毅力的',
+  '精力充沛的',
+  '健談的',
+  '機敏的',
+  '好交際的',
+  '坐懷不亂的',
+  '光明正大的',
+  '有志氣的',
+  '公正廉明的',
+  '有抱負的',
+  '開朗的',
+  '盡職認真的',
+  '寬宏大量的',
+  '可信賴的',
+  '虛懷若谷的',
+  '友好的',
+  '樂觀的',
+  '守紀律的',
+  '條理分明的',
+  '實事求是的',
+  '無拘束的',
+  '理想主義的',
+  '勇敢的',
+  '自信的',
+  '滿足的',
+  '幽默的',
+  '順服的',
+  '貫徹始終的',
+  '親切的',
+  '好動的',
+  '興趣廣泛的',
+  '肯定的',
+  '深沉的',
+  '細節的',
+  '坦率的',
+  '羞澀的',
+  '肯定的',
+  '深沉的',
+  '細節的',
+  '貫徹始終的',
+  '羞澀的',
+  '喜交朋友的',
+  '令人信服的',
+];
+
+export const separators = [
+  ({ adj, place, name }) => `${adj}${place}${name}`,
+  ({ adj, place, name }) => `${adj}✖${place}✖${name}`,
+  ({ adj, place, name }) => `來自${place}✖${adj}✖${name}`,
+  ({ adj, place, name }) => `${adj}✿${place}✿${name}`,
+  ({ adj, place, name }) => `${adj}卍${place}卍${name}`,
+];
+
+export const decorators = [
+  joinedName => `${joinedName}`,
```

Comment:
Though it's me that come up with these separators / decorators, I still think the use of separators and decorators are pretty 中二 lol

I think for separators and decorators we can duplicate identity items (such as `${adj}${place}${name}` and  `${joinedName}`) a couple of times so that most of the time there is no separators / decorators inside generated names.